spec-0007/index.md

```diff
@@ -79,7 +80,86 @@ The following users will be affected:
 
 ### Code
 
-TODO: Add example `check_random_state` implementation.
+For example, SciPy may implement this with the following decorator to support both names.
+This is implemented using:
+1. A `check_random_state` function which normalizes the either old or new input to a `Generator` or `RandomState` object.
```

Comment:
```suggestion
1. A `check_random_state` function which normalizes either old (`seed`) or new (`rng`) input to a `Generator` object.
```

Reply:
The decorator should always return a Generator, even when operating on a `RandomState`. SciPy already has example code that shows how to extract this, using the internal bit generator.